s3torchconnectorclient/python/tst/integration/test_logging.py

```diff
@@ -0,0 +1,126 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+import os.path
+import sys
+from datetime import datetime
+
+import pytest
+
+PYTHON_TEST_CODE = """
+import logging
+import os
+import sys
+
+os.environ["ENABLE_CRT_LOGS"] = "{0}"
+
+logs_dir_path = "{1}"
+if logs_dir_path != "":
+    os.environ["CRT_LOGS_DIR_PATH"] = logs_dir_path
+
+from s3torchconnector import S3MapDataset
+
+logging.basicConfig(
+    stream=sys.stdout,
+    format="%(levelname)s %(name)s %(asctime)-15s %(filename)s:%(lineno)d %(message)s",
+)
+logging.getLogger().setLevel(logging.INFO)
+
+if len(sys.argv) != 3:
+    exit("The script needs an S3 uri and a region")
+s3_uri = sys.argv[1]
+region = sys.argv[2]
+map_dataset = S3MapDataset.from_prefix(s3_uri, region=region)
+obj = map_dataset[0]
+assert obj is not None
+"""
+
+import subprocess
+
+
+@pytest.mark.parametrize(
+    "log_level, should_contain, should_not_contain",
+    [
+        (
+            "info",
+            ["INFO s3torchconnector.s3map_dataset"],
+            ["DEBUG", "TRACE"],
+        ),
+        (
+            "DEBUG",
+            [
+                "INFO s3torchconnector.s3map_dataset",
+                "DEBUG awscrt::AWSProfile",
+                "DEBUG awscrt::AuthCredentialsProvider",
+                "DEBUG awscrt::S3Client",
+            ],
+            ["TRACE"],
+        ),
+        (
+            "TRACE",
+            [
+                "INFO s3torchconnector.s3map_dataset",
+                "DEBUG awscrt::AWSProfile",
+                "DEBUG awscrt::AuthCredentialsProvider",
+                "TRACE awscrt::event-loop",
+                "TRACE awscrt::socket",
+                "TRACE awscrt::event-loop",
+            ],
+            # Python log level is set to INFO in the test script
+            ["TRACE s3torchconnector.s3map_dataset"],
+        ),
```

Comment:
Can we add another test for 'off' that shows the info for s3torchconnector still prints?

Reply:
I'll add params for it and ditch the other